set.mm

```diff
@@ -199995,6 +199996,28 @@ class of all (base sets of) groups is proper.  (Contributed by Mario
       BAHQNRSUA $.
   $}
 
+  ${
+    fveqprc.e $e |- ( E ` (/) ) = (/) $.
+    fveqprc.y $e |- Y = ( F ` X ) $.
+    $( Lemma for showing the equality of values for functions like slot
+       extractors ` E ` at a proper class.  Extracted from several former
+       proofs of lemmas like ~ zlmlem .  (Contributed by AV, 31-Oct-2024.) $)
+    fveqprc $p |- ( -. X e. _V -> ( E ` X ) = ( E ` Y ) ) $=
+      ( cvv wcel wn c0 cfv eqcomi fvprc syl5eq fveq2d 3eqtr4a ) CGHIZJJAKZCAKDA
+      KRJELCAMQDJAQDCBKJFCBMNOP $.
+  $}
+
+  ${
+    oveqprc.e $e |- ( E ` (/) ) = (/) $.
+    oveqprc.z $e |- Z = ( X O Y ) $.
+    oveqprc.r $e |- Rel dom O $.
+    $( Lemma for showing the equality of values for functions like slot
+       extractors ` E ` at a proper class.  Extracted from several former
+       proofs of lemmas like ~ resvlem .  (Contributed by AV, 31-Oct-2024.) $)
+    oveqprc $p |- ( -. X e. _V -> ( E ` X ) = ( E ` Z ) ) $=
+      ( cvv wcel wn c0 cfv eqcomi fvprc co ovprc1 syl5eq fveq2d 3eqtr4a ) CIJKZ
```

Comment:
Use eqtrid instead of syl5eq, In fveqprc, too

Reply:
Done.